packages/actions/src/debug/debugTraceCallProcedure.spec.ts

```diff
@@ -14,6 +14,29 @@ describe('debugTraceCallJsonRpcProcedure', () => {
 		assert(createdAddress, 'Contract deployment failed')
 		const contract = SimpleContract.withAddress(createdAddress)
 
+		expect(
+			await procedure({
+				jsonrpc: '2.0',
+				method: 'debug_traceCall',
+				params: [
+					{
+						to: contract.address,
+						data: encodeFunctionData(contract.write.set(42n)),
+					},
+				],
+				id: 1,
+			}),
+		).toMatchSnapshot()
+	})
+
+	it.todo('should trace a transaction and return the expected result with callTracer', async () => {
```

Comment:
```suggestion
	it('should trace a transaction and return the expected result with callTracer', async () => {
```
is this intentional?